src/tvheadend/HTSPDemuxer.cpp

```diff
@@ -297,7 +297,13 @@ bool HTSPDemuxer::IsTimeShifting() const
 
 bool HTSPDemuxer::IsRealTimeStream() const
 {
-  return m_subscription.IsActive();
+  if (!m_subscription.IsActive())
+    return false;
+
+  if (m_subscription.GetSpeed() != SPEED_NORMAL)
```

Comment:
this is wrong. where did you get this from?

Reply:
Ooops, this was there in the code before the  last change of this method and slipped in accidentally. I will remove that.